stdlib/builtin/binding.rbs

```diff
@@ -31,7 +31,7 @@
 # 
 # [Binding](Binding) objects have no class-specific
 # methods.
-class Binding < Object
+class Binding[T] < Object
```

Comment:
I think we need a variance annotation here `out T`.

Reply:
I didn't know the variance, thank you for the comment.